src/sparql/StardogSparqlParser.ts

```diff
@@ -78,6 +78,114 @@ export class StardogSparqlParser extends BaseSparqlParser {
     ]);
   });
 
+  TriplesSameSubject = this.OVERRIDE_RULE('TriplesSameSubject', () => {
```

Comment:
For review, compare: https://github.com/stardog-union/stardog/pull/7892/files#diff-5c15a51cf8dbf56652a3f3aa526f5137